edit-content.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: docs
+title: "Documentation Site Guide | Edit Content"
+---
+
+## Edit Content
+
+To edit documentation that has already been added to GitHub and is on the website, work in the `development` branch of the GitHub repository for that set of documentation.
+
+1. Commit changes with descriptive commit messages.
```

Comment:
```suggestion
1. Commit changes with descriptive commit messages that describe the changes instead of the default "update [file]" message provided in GitHub. For example, messages like "add config values," "fix typo," or "adjust heading levels" clearly communicate what has changed.
```
Adding same language as on the "Create docs page" since I don't think we can assume folks will read both pages.